lib/Adafruit_BNO055/Adafruit_BNO055.h

```diff
@@ -307,10 +307,13 @@ class Adafruit_BNO055 : public Adafruit_Sensor
     void  setSensorOffsets(const adafruit_bno055_offsets_t &offsets_type);
     bool  isFullyCalibrated(void);
 
+    // make public to write in imuService
+    bool  write8  ( adafruit_bno055_reg_t, byte value );
```

Comment:
Maybe a better way would be to add a method `setAxisRemapConfig()`. This would respect the principles of encapsulation better.

Reply:
May be we should write this to Adafruit, to let them make it, so we can use the original library?

Reply:
That would be the right thing to do indeed :) Maybe in another PR ;) Let's tackle one problem at a time.